### DISCUSSION TITLE
VcDrawings绘制出矩形后，如何得到它的实例对象，快速获取矩形的四个点的坐标？

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#615] [Q&A] [answered]
VcDrawings绘制出矩形后，如何得到它的实例对象，快速获取矩形的四个点的坐标？如图，用下面的方法似乎获取不到该矩形的坐标点相关参数：
![bae91947-21a7-4e84-bbd2-5fdc313bcddf](https://github.com/user-attachments/assets/975f5ce5-0c47-4eb4-b1af-828fd1a9d295)
nan'da
难道只能自己去viewer的primitives里面遍历找到这个矩形对象吗？

**Comment by zouyaoji** [comment, marked as answer]
通过模板引用来获取到绘制的数组，比如
drawingsRef.getDrawingActionInstances()[3].cmpRef.value.renderDatas
绘制的对象信息都在这里面。
